### PR TITLE
Add crisis acquisition support banner to homepage

### PR DESCRIPTION
**Pull request by igorkorenfeld**
Relates to issue(s) #3155.

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/BRANCH_NAME)

[:sunglasses: PREVIEW](https://federalist-d703a102-c627-4805-8934-78f5bf4c97da.app.cloud.gov/site/igorkorenfeld/18f.gsa.gov/)

Changes proposed in this pull request:
- Add announcement banner on home page with link to the new rapid IT support section on the How We Work page

Checklist:
- [x] All images being added have been optimized **_--> [learn more](https://18f.gsa.gov/styleguide/images) about how to optimize images <--_**

/cc @Dahianna @erinzstrenio @michelle-rago @StephanieRivera

### CODE REVIEW

File: _sass/_templates/home.scss [hunk 1]
```diff
@@ -87,3 +87,21 @@ section.posts_feature h2 a {
...
+.announcement__body {
+  background-image: url('../img/page-feature/info.svg');
+  background-repeat: no-repeat;
+  background-size: $icon-size;
+  background-position: 0 $spacing-md-small;
+  padding: $spacing-md-small 0;
+  padding-left: $icon-size + $spacing-md-small;
+}
+
+.announcement__text{
```
[the remark this answers is not in the document]
Comment (igorkorenfeld) [reply _sass/_templates/home.scss]: Added!

File: _sass/_templates/home.scss [hunk 2]
```diff
@@ -87,3 +87,21 @@ section.posts_feature h2 a {
   color: #1F2E4A;
   text-decoration: none;
 }
+
+//New Feature announcement
+.announcement{
```
[the remark this answers is not in the document]
Comment (igorkorenfeld) [reply _sass/_templates/home.scss]: Added!

**Comment by michelle-rago** [comment]
Looks good to me!

**Comment by Dahianna** [comment]
Once all check are complete. I'll go ahead and merge

**Pull request merged**